docs/cookbook/modifying-a-template-in-administration.md

```diff
@@ -0,0 +1,133 @@
+# Modifying a Template in Administration
+
+In this tutorial, we'll show you how to proceed if you need to modify a twig template in the administration.
+We will demonstrate this procedure with the example How to display a product transfer status on the product detail page in the administration.
+
+## Adding the information into the template
+
+This tutorial assumes that the product already contains the transfer status attribute.
+If you need to extend the product with a new attribute, see the tutorial [Adding new attribute to an entity](./adding-new-attribute-to-an-entity.md).
+
+Modifying the bundle templates that are located in the vendor can be done via overriding these templates.
+
+*If you need to extend or modify the form itself, it is not necessary to use the overriding and consequently to lose an upgradeability.
+For the extensions of the forms, see [Adding new attribute to an entity](./adding-new-attribute-to-an-entity.md#enable-administrator-to-edit-the-extId-field).*
```

Comment:
[Adding new attribute to an entity] - this is confusing for me, you talk about extending the forms but it leads to article with big header "Adding new attribute to an entry" it could be something like "For the extension of the form, see [Ediation of new attribute in Administration]

Reply:
Modified